index.js

```diff
@@ -1018,7 +1018,7 @@ export var detectedKeywords = null;
  * This also replicates JQuery `$(document).ready()`, with added protection for
  * usage of `async` -- the DOM ready event can fire before the script is loaded..
  */
-if (check_dependencies()) {
+if (check_dependencies() && !window.ethicalads) {
```

Comment:
I'm not sure if it would make a big difference here, but `globalThis` is a great way to refer to the global object in multiple contexts -- that is `globalThis === window` in the browser and `globalThis === global` within `node`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

But using web-test-runner the usage is probably almost strictly in the browser anyways.

Reply:
I'm a little hesitant to use `globalThis` due to support. Do you think this is widely supported enough? https://caniuse.com/mdn-javascript_builtins_globalthis